var/spack/repos/builtin/packages/py-biopython/package.py

```diff
@@ -28,6 +28,7 @@ class PyBiopython(PythonPackage):
     version("1.65", sha256="6d591523ba4d07a505978f6e1d7fac57e335d6d62fb5b0bcb8c40bdde5c8998e")
 
     depends_on("c", type="build")  # generated
+    depends_on("libxcrypt", when="%apple-clang")
```

Comment:
```suggestion
```

>  fixing/diagnosing this would basically the responsibility of the apple-clang setup, not of a user package like biopython that should not have anything to do with what includes LLVM has.

I agree with this assessment. I did not see anything in the `biopython` repository indicating that this is a dependency of the package.